data/darktableconfig.xml.in

```diff
@@ -254,9 +254,9 @@
   <dtconfig>
     <name>opencl_mandatory_timeout</name>
     <type min="0">int</type>
```

Comment:
BTW, does it makes sense to have a min timeout of 0 for this setting?

Reply:
Given the loop structure the `min=0` would AFAIK simply never lock? Maybe it'd be better to have it min 1...

Reply:
Yes, maybe min=100 (half a second?).

Reply:
OK. Done!